version.tf

```diff
@@ -9,5 +9,12 @@ terraform {
     null = {
       version = ">= 3.2.1"
     }
+    time = {
+      version = "0.9.1"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.16.1"
+    }
```

Comment:
These are listed as required providers. Does that mean ALL the examples MUST be updated to provide them? Does that also mean ALL consumers (external modules, customer uses) must also be update to provider them? I thought it did, but yet I am not seeing the required provider for null being treated this way.

Reply:
should of used range here, and not locked into exact version. Created internal issue to fix